crates/connectors/ndc-postgres/src/schema.rs

```diff
@@ -479,7 +499,7 @@ fn make_procedure_type(
     scalar_types
         .entry("int4".to_string())
         .or_insert(models::ScalarType {
-            representation: None,
+            representation: Some(models::TypeRepresentation::Integer),
```

Comment:
This is really something of an edge case. I wonder if we actually need to register `int4`, as in, I wonder if there actually is a functional difference between `"scalarTypes":{"int4": {"aggregateFunctions": {}, "comparisonOperators": {}}` and `"scalarTypes": {}`.

At any rate, it is a curious detail that this edge case gives a different result than the common case where `int4` is defined, but we don't yet assign type representations to anything but enums.

Reply:
Yes, we definitely need it. I think I added it because a test failed. Try and see :)

The comments about this are not visible in this diff by default, adding it here:

```
    // If int4 doesn't exist anywhere else in the schema, we need to add it here. However, a user
    // can't filter or aggregate based on the affected rows of a procedure, so we don't need to add
    // any aggregate functions or comparison operators. However, if int4 exists elsewhere in the
    // schema and has already been added, it will also already contain these functions and
    // operators.
```